sky/provision/kubernetes/instance.py

```diff
@@ -566,7 +615,7 @@ def _create_pods(region: str, cluster_name_on_cloud: str,
                        'For more details, refer to https://skypilot.readthedocs.io/en/latest/reference/config.html')  # pylint: disable=line-too-long
 
     needs_gpus = (pod_spec['spec']['containers'][0].get('resources', {}).get(
-        'limits', {}).get('nvidia.com/gpu', 0) > 0)
+        'limits', {}).get(kubernetes_utils.GPU_RESOURCE_KEY, 0) > 0)
     if nvidia_runtime_exists and needs_gpus:
         pod_spec['spec']['runtimeClassName'] = 'nvidia'
```

Comment:
Does gcp has default runtime for TPU? Should we add a todo here to automatically set runtime to tpu in the future?

Reply:
@cblmemo GCP has default runtime for GKE, which seems to be containerd, https://cloud.google.com/kubernetes-engine/docs/how-to/migrate-containerd#overview.

Reply:
Sounds great! Maybe add this to the comment ;)